src/xdp/trn_transit_xdp.c

```diff
@@ -175,44 +176,49 @@ static __inline int trn_process_inner_ip(struct transit_packet *pkt)
 		flow->dport = pkt->inner_udp->dest;
 	}
 
-	/* Modify inner EtherHdr */
-	trn_set_dst_mac(pkt->inner_eth, ep->mac);
-
-	/* Keep overlay header, update outer header destinations */
-	trn_set_src_dst_ip_csum(pkt->ip, pkt->ip->saddr, ep->hip, pkt->data_end);
-	trn_set_dst_mac(pkt->eth, ep->hmac);
-
 	/* Generate Direct Path request */
-	pkt->fctx.opcode = XDP_FLOW_OP_ENCAP;
+	pkt->fctx.opcode = bpf_htonl(XDP_FLOW_OP_ENCAP);
```

Comment:
Very small optimization, perhaps we can store these constants as network byte order, saving an extra op in the datapath .

Reply:
For constant, no need as they are done at compile time, not run time, and this will make it better readable. Actually we define the VxLAN/geneve port in network order is not prefered.